### PR TITLE
[feature/#39] Generate player average score

### PR DESCRIPTION
**Pull request by AdamJ335-T2**

### CODE REVIEW

File: src/app/model/group/player.py [hunk 1]
```diff
@@ -8,12 +8,14 @@ class Player(Base):
...
 
     @classmethod
     def from_dict(cls, dic):
         return cls(
             name=dic["name"],
             streak=dic["streak"],
-            score=dic["score"]
+            score=dic["score"],
+            played=dic["played"]
         )
```
Comment (jrsmth) [inline src/app/model/group/player.py:21]: How about `player.games` and `player.score`?

Then we just return the average at presentation time - i.e `player.getAverageScore()` -> `self.score / self.games`

Comment (AdamJ335-T2) [reply src/app/model/group/player.py:21]: I like that approach. Makes it a lot more usable. Good idea

File: documentation/CHANGELOG.md [hunk 2]
```diff
@@ -7,10 +7,14 @@
 - `0.3.x` Scroll Command
 - `0.4.x` Presentation 
 - `0.5.x` Reactions
-- `1.0.0` App Distribution
+- `1.0.x` App Distribution
 
 # Releases
 <!-- @LatestFirst -->
```
Comment (jrsmth-tier2) [inline documentation/CHANGELOG.md:13]: Is there an empty line here (should be)

File: src/app/model/group/player.py [hunk 3]
```diff
@@ -8,12 +8,14 @@ class Player(Base):
     """ Tracked player information """
     name: str
     streak: int
-    score: int
+    score: float
```
Comment (jrsmth-tier2) [inline src/app/model/group/player.py:11]: `int` (Already discussed)

File: documentation/CHANGELOG.md (hunk 2 above)
Comment (AdamJ335-T2) [reply documentation/CHANGELOG.md:13]: I am updating the changelog now

File: src/app/archbishop/archbishop.py [hunk 4]
```diff
@@ -57,6 +71,10 @@ def handle_waffle(message, say):
         say(response)
         return Response(messages.load("event.request.handled"), status=200)
 
+    def get_avg_score(player):
+        # Return average score to 2 decimal places
+        return round(player.score / player.games, 2)
+
```
Comment (jrsmth-tier2) [inline src/app/archbishop/archbishop.py:77]: How about putting this on the player itself?

`player.average()` -> `return round(self.score / self.games, 2)`

Comment (AdamJ335-T2) [reply src/app/archbishop/archbishop.py:77]: That makes infinitely more sense. Im clearly not on the ball this morning!

Comment (AdamJ335-T2) [reply src/app/archbishop/archbishop.py]: Updated now. Please Review

File: src/app/archbishop/archbishop.py [hunk 5]
```diff
@@ -22,6 +22,19 @@ def hello():
...
+        scroll_list = redis.get_complex(group_id, Group).scroll
+
+        # Append list of Scroll entries
+        scroll_entry_message = ""
+        for i in range(len(scroll_list)):
+            scroll_entry = scroll_list[i]
+            scroll_entry_message += messages.load_with_params("command.scroll.entry", [str(i+1), scroll_entry.name, str(scroll_entry.streak), str(scroll_entry.date)])
+
+        return Response(scroll_entry_message, status=200)
+
```
Comment (jrsmth-tier2) [inline src/app/archbishop/archbishop.py:37]: What do you think about this?
```
@archbishop.route("/scroll", methods=['POST'])
def unroll():
    """ Handle slack command for scroll information """
    group =  redis.get_complex(request.form["team_id"], Group) 
    formatted_scroll = ""
    position = 0
    for record in group.scroll:
        position += 1
        formatted_scroll += messages.load_with_params(
            "command.scroll.entry", 
            [str(position), record.name, str(record.streak), str(record.date)]
        )
    return Response(formatted_scroll, status=200)

File: documentation/CHANGELOG.md [hunk 6]
```diff
@@ -38,3 +49,4 @@
 [0.1.0]: https://github.com/jrsmth/waffle-bot/compare/0.0.0...0.1.0
 [0.1.1]: https://github.com/jrsmth/waffle-bot/compare/0.1.0...0.1.1
 [0.2.0]: https://github.com/jrsmth/waffle-bot/compare/0.1.1...0.2.0
+[0.3.0]: https://github.com/jrsmth/waffle-bot/compare/0.2.0...0.3.0
```
Comment (jrsmth-tier2) [inline documentation/CHANGELOG.md:52]: Good stuff, what about `[0.4.0]` tag?

**Comment by AdamJ335-T2** [comment]
Reviewed by James and Hayden
closes #39

**Pull request merged**